schema/filter_artworks.js

```diff
@@ -48,6 +48,9 @@ const FilterArtworks = {
     dimension_range: {
       type: GraphQLString,
     },
+    extra_aggregation_gene_ids: {
```

Comment:
Assuming we have a list of genes we want aggregations for...we can just make the metaphysics query with them.

If you're trying to _get_ the list of genes, and then also get the aggregations for those genes all at once- I think that's getting ahead of ourselves. We haven't even settled on the implementation of those category/related category things. 

If we do the featured link thing....those would need to be fetched before asking for the aggregations, and then plugged into the filter query. But if we just stick that list in S3/JSON somewhere...we'd have them ahead of the time and can just pass it in.